plugin.yaml

```diff
@@ -348,6 +352,34 @@ node_types:
         resize:
           implementation: vsphere.vsphere_storage_plugin.storage.resize
 
+  cloudify.vsphere.nodes.ISO9660:
```

Comment:
I think this should probably be named CDROM or something like that- I'm not convinced the ISO standard number is going to be as widely recognised.

Reply:
What do you think about CloudInitCDROM? 

Because in reality, it's not so cdrom, more cdrom with all tags and format flags that will be enough to use as cloudinit.

Reply:
If that's explicitly all we're supporting then yes, that makes sense.

In that case, you might want to make the example in the 'file list' help (next comment) match the expected structure that cloudinit wants?